MicrosoftTeams.schema.json

```diff
@@ -66,6 +66,12 @@
             "type": "object",
             "additionalProperties": false,
             "properties": {
+                // the below property can be avoided by reducing the size of the short property.
```

Comment:
The addition of tiny property can be avoided by reducing the size of the short property. Ideally an app nam will be in the range of upto 12 chars to be shown in the app bar. developer should also account for localisation/ other languages where the character width might be large.

Reply:
We can't make existing app properties more restrictive in our schema. For back compatibility reasons, we can only create new properties or relax requirements on existing ones. I think we should just introduce this new property for apps that want to look great on Mobile. If the property isn't available for an app then you guys can just take the short name and truncate it.

Reply:
We totally could make the properties more restrictive.. it would just mean new submissions would have to change when they update their manifest... not a big deal... However for this situation I think adding the tiny name makes sense because we don't want people to have to change their app name.

Reply:
What I mean is that this schema gets reflected in our Settings Store which enforces it on all app definitions written in there. If we made certain field more restrictive then we wouldn't be able to reflect these changes in our Settings Store schema since there might already be existing app definitions that will be non-conformant.

Reply:
Oh yes, I see what you are saying @ydogandjiev. But we could make this schema more restrictive and just not reflect that in Settings Store. new apps would effectively have the shorter name because the schema wouldn't allow them to submit longer names but the settings store still would have to technically allow up to ~30

Reply:
I think it makes sense to add the Tiny field... We show the short name everywhere and it would be weird to force developers to shorten their app name to 10 characters everywhere just so they can work well in mobile.
alternatively, we could use the long name in places we use the short name now (since most apps have the same long name as short name) and force short name to be only shown in this mobile case... (at the moment we do not use long name anywhere).
I'm ok with either option... however instead of just advising devs to make their app name shorter... I would rather have a strict schema enforcement